### PR TITLE
Add a PR template

### PR DESCRIPTION
**Pull request by cjellick**
Signed-off-by: Craig Jellick <craig@acorn.io>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+-->
+
+#### Verification and Testing ####
+
+<!-- How can the changes be verified? Please provide whatever additional information necessary to help verify the proposed changes. -->
+<!-- Are there automated tests to cover your changes? -->
+
+#### Linked Issues ####
+
+<!-- Link any related issues, pull-requests, or commit hashes that are relevant to this pull request. If you are opening a PR without a corresponding issue please consider creating one first, at https://github.com/k3s-io/k3s/issues . A functional example will greatly help QA with verifying/reproducing a bug or testing new features. -->
```
Comment (thedadams) [inline .github/PULL_REQUEST_TEMPLATE.md:27]:
```suggestion
<!-- Link any related issues, pull-requests, or commit hashes that are relevant to this pull request. If you are opening a PR without a corresponding issue please consider creating one first, at https://github.com/acorn-io/acorn/issues . A functional example will greatly help QA with verifying/reproducing a bug or testing new features. -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!-- HTML Comments can be left in place or removed. -->
```
Comment (tylerslaton) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: nit: Necessary?
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+-->
+
+#### Proposed Changes ####
+
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. -->
+
+#### User-Facing Change? ####
+<!--
+Does this PR introduce a user-facing change? If no, just write "NONE" in the release-note block below.
```
Comment (tylerslaton) [inline .github/PULL_REQUEST_TEMPLATE.md:16]: nit: I'm not seeing the release-note block that this reference.

**Review by tylerslaton — APPROVED** [review]
Nice! We should also add some issue templates.

**Comment by ibuildthecloud** [comment]
Does this create a form in github?  I've seen other projects where it's a form and not just have to edit markdown.

**Comment by cjellick** [comment]
Thanks @tylerslaton was trying to find an example

**Comment by cjellick** [comment]
ah @ibuildthecloud @tylerslaton, from the doc Tyler shared, PR templates dont support form fields, just issue templates, which makes sense...I couldn't really square how form fields would work well in a tempalte.

Nevertheless, I updated the whole template a bit.

**Comment by cjellick** [comment]
> Nice! We should also add some issue templates.

Yeah, i'll ahve someone do that next. I think we might want to consider slightly better issue reporting tools as part of that (like gather logs and whatnot).

**Comment by tylerslaton** [comment]
> Yeah, i'll ahve someone do that next. I think we might want to consider slightly better issue reporting tools as part of that (like gather logs and whatnot).

Makes sense, we'll want that form layout there in that case which we can likely setup to encourage the inclusion of debugging.

**Review by tylerslaton — APPROVED** [review]
Re-approving.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] All relevant issues are referenced in this PR
+- [ ] The title of this PR would make a good line in the Release Note's Changelog
+- [ ] Commits are signed-off using `git commit -s`
+- [ ] Automated tests added to cover the changes. If tests couldn't be added, an explanation is provided in the Verification and Testing section.
+
+
+#### Proposed Changes
+_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request_
+
+
```
Comment (tylerslaton) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: very small and unimportant nit
```suggestion
#### Proposed Changes
_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request_

```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+#### User-Facing Change? ####
+_Does this PR introduce a user-facing change either in the API or CLI or in functionality?
```
Comment (tylerslaton) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
_Does this PR introduce a user-facing change in functionality, the API or the CLI?_
```

**Review by tylerslaton — APPROVED** [review]

**Review by jsilverio22 — APPROVED** [review]

**Review by StrongMonkey — APPROVED** [review]

**Review by thedadams — APPROVED** [review]

**Pull request merged**